src/stylesheets/core/_variables.scss

```diff
@@ -1142,3 +1142,9 @@ $spacing-small: 1rem;
 $spacing-md-small: 1.5rem;
 $spacing-medium: 2rem;
 $spacing-large: 3rem;
+
+$theme-spacing-x-small: 0.5; // was 0.5rem
```

Comment:
@thisisdano If we want to use tshirt sizing here how would that look?

perhaps:
```
theme-spacing-xs
theme-spacing-sm
theme-spacing-md
theme-spacing-lg
```

Reply:
Let's go with this — I have some qualms about imposing a project-level tshirt spacing scale, but let's lay them aside for right now